eventstore/tasks.py

```diff
@@ -319,6 +323,76 @@ def update_rapidpro_contact_error_timestamp(context):
     )
 
 
+@app.task(
+    autoretry_for=(RequestException, ConnectionException),
```

Comment:
We're not making any HTTP requests here, so these are probably not useful exceptions to retry on. We should look at what exceptions `pysftp` throws for things like connection errors, that we could retry on.

Reply:
One of them is pysftp.ConnectionException. https://pysftp.readthedocs.io/en/release_0.2.8/pysftp.html